packages/provider/src/transports/wallet-request-handler.ts

```diff
@@ -831,13 +826,17 @@ export class WalletRequestHandler implements ExternalProvider, JsonRpcHandler, P
 }
 
 export interface WalletUserPrompter {
+  getDefaultNetwork(): number
```

Comment:
I think we should use the term chainId. ie.`getDefaultChainId` to avoid any disambiguation with NetworkConfigs